src/components/Forecast/index.tsx

```diff
@@ -51,28 +65,29 @@ function Forecast({ data }: Props) {
           <Graph
             ContainerInfo={containerData}
             temperature={data.map((each) => each.dayTemp)}
-            FromTop="30%"
+            fromTop="30%"
             positioning="dayTime"
           />
           {/* for nightTime */}
           <Graph
             ContainerInfo={containerData}
             temperature={data.map((each) => each.nightTemp)}
-            FromTop="55%"
+            fromTop="55%"
             positioning={undefined}
           />
         </>
       )}
 
-      {data.map((bud) => (
-        <S.EachDay key={bud.day}>
-          <S.BigFont>{bud.day}</S.BigFont>
-          <S.SmallFont>{bud.date}</S.SmallFont>
-          <S.Images src={bud.weather} alt="weather" />
+      {data.map((day) => (
+        <S.EachDay key={day.time}>
+          <S.BigFont>{day.date}</S.BigFont>
+          <S.SmallFont>{day.time}</S.SmallFont>
+
+          <S.Images src={getWeatherIcon(day.weather)} alt="weather" />
           <S.GraphSpace ref={childRef}></S.GraphSpace>
           <S.GraphSpace></S.GraphSpace>
-          <S.Images src={bud.weatherNight} alt="weatherNight" />
-          <S.SmallFont>{bud.wind}</S.SmallFont>
+          <S.Images src={moon} alt="weatherNight" />
+          <S.SmallFont>{day.wind}</S.SmallFont>
```

Comment:
Update this information somehow so that it is clear on the UI that this is the wind)